docs/content/en/blog/releases/v0.26.0.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Release v0.26.0"
+linkTitle: "Release v0.26.0"
+date: 2022-02-21
+description: >
+ Release v0.26.0
+---
+
+## Changelog since v0.25.1
+
+### Notable Changes
+
+* Add ability to specify api-versions for helm by using pipecd application configurations ([#3259](https://github.com/pipe-cd/pipecd/pull/3259))
+* Add ability to specify kube-version for helm by using pipecd application configurations ([#3261](https://github.com/pipe-cd/pipecd/pull/3261))
+* Enable filter notifications based on application/deployment's labels ([#3241](https://github.com/pipe-cd/pipecd/pull/3241))
+* Enable to use `pipecd.dev/sync-by-replace: "enabled"` annotation in Kubernetes manifests to use `kubectl replace/create` instead of `kubectl apply` while applying it ([#3237](https://github.com/pipe-cd/pipecd/pull/3237))
+* The env-id and env-name were removed from `pipectl application list` command ([#3294](https://github.com/pipe-cd/pipecd/pull/3294))
+* Fix wrong summary result for Kubernetes application in case the image version was updated ([#3285](https://github.com/pipe-cd/pipecd/pull/3285))
+* Remove env-id option from pipectl's add-application command ([#3276](https://github.com/pipe-cd/pipecd/pull/3276))
+* Avoid error from actions-plan-preview when sending a too long message on GitHub ([#3253](https://github.com/pipe-cd/pipecd/pull/3253))
+* Fix unable to register event in case of using MySQL as datastore ([#3245](https://github.com/pipe-cd/pipecd/pull/3245))
+* Remove Environment concept from web view ([#3217](https://github.com/pipe-cd/pipecd/pull/3217))
```

Comment:
should we move those kinds of changes related to Environment as `breaking change` and mention notes to migrate (env => label env blog: https://pipecd.dev/blog/2022/02/10/february-2022-update/#environment-and-deployment-configuration-file-were-deprecated) 🤔

Reply:
Right. Let me do it.

Reply:
@khanhtc1202 Addressed by https://github.com/pipe-cd/pipecd/pull/3296/commits/2a226e6f3b8bbaefd9de31d23d1ffc4b0d95f1b0.